### ISSUE TITLE
Combine questions about the member code and workshop costs

### ISSUE THREAD

**Issue by maneesha** [carpentries/amy#2583]
On the workshop request form, with changes for the membership code, we ask for the member code and then ask again if costs don't apply because they are a member. Consider combining these questions so the member code question is available only if they check off they are a member and in that case, is required.  

So this question...
![image](https://github.com/carpentries/amy/assets/829690/fab27983-08a3-4d20-af3b-5a236ae5c3f5)

...should appear only if and should be required if the member option is checked here:
![image](https://github.com/carpentries/amy/assets/829690/4b3776f6-85a6-4f2e-99cd-6ffeac2e9cd8)

Perhaps this can be accomplished doing something similar to the Instructor Training form where we begin with a question about their member status.

Tagging @danielle06 @sheraaronhurt @elletjies for feedback.

**Comment by maneesha** [comment]
Flow of what this could look like:


> Please review [The Carpentries website workshop fee listing.](https://carpentries.org/workshops/#workshop-cost) before completing this form.  
> * Are you requesting this workshop through a Carpentries member organisation? 
>     * Yes, I am with a Member organisation so the workshop fee does not apply (instructor travel costs will still apply for in-person workshops).
>         * Please enter your Membership registration code.
>     * No 
>         * I am with a government site, university, or other nonprofit. I understand the workshop fee as listed on The Carpentries website and agree to follow through on The Carpentries invoicing process.
>         * I am with a corporate or for-profit site. I understand the costs for for-profit organisations are four times the price for not-for-profit organisations.
>         * I am requesting financial support for the workshop fee (instructor travel costs will still apply for in-person workshops)

**Comment by elletjies** [comment]
@maneesha, if I understand correctly, this will be fewer clicks, right?

**Comment by maneesha** [comment]
The number of clicks will depend on what the UI ends up looking like but this should reduce redundancy.

**Comment by danielle06** [comment]
I agree with Sher, I really like the workflow you have proposed here. Thanks, @maneesha!

**Comment by maneesha** [comment]
Thanks @danielle06  and @sheraaronhurt .

@elichad  please note this should be done along with #2582 since that is about the language used in the for-profit organization's question.
